packages/manager/src/components/ChatUrDocs/Chat/Chatbot.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from 'react';
+import { Card, CardBody } from '@patternfly/react-core';
+import ChatbotMessage from './ChatbotMessage';
+import ChatbotInput from './ChatbotInput';
+import '../ChatUrDocs.css';
+import MarkdownRenderer from './MarkdownRenderer';
+import { marked } from 'marked';
+
+interface ChatMessage {
+  content: string;
+  isUserMessage: boolean;
+}
+
+interface ChatbotProps {
+  botName: string;
+}
+
+const Chatbot: React.FC<ChatbotProps> = ({ botName }) => {
+  const [chatMessages, setChatMessages] = useState<ChatMessage[]>([]);
+
+  useEffect(() => {
+    // Send initial message when component mounts
+    sendInitialMessage();
+  }, []);
+
+  const sendInitialMessage = () => {
+    const initialMessage =
+      'Hi! 😊 I’m ' + botName + ', I’m here to help you with any questions or issues.';
+    // Display the initial message in the chat
+    appendMessage(initialMessage, false);
+  };
+
+  const appendMessage = (message: string, isUserMessage: boolean) => {
+    // Update the state to include the new message
+    setChatMessages((prevMessages) => [...prevMessages, { content: message, isUserMessage }]);
+  };
+
+  const handleSendMessage = async (userMessage: string) => {
+    // Display the user's message in the chat
+    appendMessage(userMessage, true);
+
+    // Send the user's question to the backend
+    sendUserQuestion(userMessage);
+  };
+
+  const sendUserQuestion = async (userQuestion: string) => {
+    const API_ENDPOINT = 'https://chaturdocs-api.usersys.redhat.com/chaturdocs/rest/search';
```

Comment:
Please mask this url, this is an internal URL, this should be ideally from env and maybe a axios config might help

Reply:
done